tests/unit/lastuser_core/test_session_UserSession.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+
+from lastuserapp import db, init_for
+import lastuser_core.models as models
+from .test_db import TestDatabaseFixture
+from datetime import datetime as dt
+from coaster.utils import buid
+from datetime import datetime, timedelta
+from flask.testing import FlaskClient
+from flask import url_for
+from bs4 import BeautifulSoup
+
+
+class TestUser(TestDatabaseFixture):
+
+    def test_UserSession_init(self):
+        """Test to verify the creation of UserSession instance"""
+        result = models.UserSession()
+        self.assertIsInstance(result, models.UserSession)
```

Comment:
`result` _has_ to be an instance of `UserSession`, unless `UserSession` is a function, which is unlikely. So, I'm not sure how this test case is useful.